src/cdk/stack.test.ts

```diff
@@ -0,0 +1,118 @@
+import { MiraServiceStack, MiraStack } from './stack'
+import { MiraApp } from './app'
+import { MiraConfig } from '../config/mira-config'
+import { Stack, CfnOutput, Tag } from '@aws-cdk/core'
+
+jest.mock('@aws-cdk/core', () => ({
+  ...jest.requireActual('@aws-cdk/core'),
+  CfnOutput: jest.fn()
+}))
+
+MiraConfig.getEnvironment = jest.fn().mockReturnValue({
+  env: {
+    account: '101259067028',
+    region: 'eu-west-1'
+  },
+  profile: 'mira-dev',
+  name: 'default'
+})
+
+describe('MiraServiceStack', () => {
+  const app = new MiraApp()
+
+  it('applyPolicies calls applyAspect', () => {
+    const miraServiceStackInstance = new MiraServiceStack(app, 'env')
+    miraServiceStackInstance.node.applyAspect = jest.fn()
+    expect(miraServiceStackInstance.applyPolicies([])).toEqual(undefined)
+    expect(miraServiceStackInstance.node.applyAspect).toBeCalledTimes(1)
+  })
+
+  it('has app initialized', async () => {
+    const miraServiceStackInstance = new MiraServiceStack(app, 'env', 'sufix')
+    expect(await miraServiceStackInstance.initialize()).toEqual(undefined)
+    expect(await miraServiceStackInstance.initialized).toBe(undefined)
+  })
+})
+
+describe('MiraStack', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  const stack = new Stack()
+
+  it('has app initialized', async () => {
+    const miraStackInstance = new MiraStack(stack, 'Default', {})
+    expect(await miraStackInstance.initialize()).toEqual(undefined)
+    expect(await miraStackInstance.initialized).toBe(undefined)
```

Comment:
Because the [resolved promise](https://github.com/nearform/mira/blob/7aefd7690b80f4738ee544b84d433503564cb467/src/cdk/stack.ts#L40) doesn't return anything this "undefined" test is actually testing if there was an error or no

Reply:
Please can you put that as a comment in the test

Reply:
Done